src/linker/Linker/MessageContainer.cs

```diff
@@ -7,6 +7,8 @@
 
 namespace Mono.Linker
 {
+	// See a list of error and warning codes at https://github.com/mono/linker/blob/master/src/linker/ErrorAndWarningCodes.md
```

Comment:
Can you please put this into the XML comments on the specific methods? Basically we want people to know about this list whenever they're reporting warning/error.

Reply:
The ask was to add XML style comments onto the public methods which create the messages. So on `MessageContainer.CreateErrorMessage` and so on. The comment should look like
```C#
This should be an XML comment:
/// <summary>Creates an error message</summary>
/// <param name="text">The content of the error message</param>
/// <param name="code">The code of the error - please see https://github.com/mono/linker/blob/master/src/linker/ErrorAndWarningCodes.md for the list of error/warning codes and possibly add a new one there.</param>
public static MessageContainer CreateErrorMessage (string text, int code, ...)
```

When written like this VS and VS Code will show this in a nice way in the UI in intellisense.

Reply:
Referring tot he list in places where we use the codes is not really needed - I would remove those comments.

The desire is such that if somebody new wants to write an error message and types the `MessageContainer.CreateErrorMessage(` into the IDE it will lpopup descriptive text which will guide them how to create the best error message possible - including making sure to use the right code and to register it in the md file when necessary.